src/stories/index.stories.ts

```diff
@@ -0,0 +1,47 @@
+import { storiesOf } from '@storybook/angular';
+import { action } from '@storybook/addon-actions';
+import { linkTo } from '@storybook/addon-links';
+
+import { Welcome, Button } from '@storybook/angular/demo';
+
+storiesOf('Welcome', module).add('to Storybook', () => ({
```

Comment:
Leftover from Storybook generation?

Reply:
Yep. That whole directory an probably go away.